rabata/resource_rabata_s3_bucket_object.go

```diff
@@ -330,6 +330,7 @@ func resourceRabataS3BucketObjectUpdate(ctx context.Context, d *schema.ResourceD
 	key := d.Get("key").(string)       //nolint:forcetypeassert
 
 	if d.HasChange("acl") {
+		//nolint:forcetypeassert
```

Comment:
Let put nolint only to required line

Reply:
![image](https://github.com/user-attachments/assets/be2aef16-7a53-4e44-a593-98c8b104720a)

I've tried, won't work that way